### PR TITLE
Added Exporter and Notifications categories and removed them from Data Collection

### PR DESCRIPTION
**Pull request by sashwathn**
Please review and approve

### CODE REVIEW

**Review by thiagoftsm — DISMISSED** [review]

**Review by andrewm4894 — DISMISSED** [review]

File: integrations/categories.yaml [hunk 1]
```diff
@@ -413,3 +401,27 @@
...
+  description: "Exporter Integrations"
+  most_popular: true
+  priority: 4
+  children:
+- id: notify
+  name: notifications
+  description: "Notification Integrations"
+  most_popular: true
+  priority: 3
+  children:
```
Comment (andrewm4894) [inline integrations/categories.yaml:415]:
```suggestion
  children:
```

File: integrations/categories.yaml [hunk 2]
```diff
@@ -413,3 +401,27 @@
       most_popular: false
       priority: -1
       children: []
+- id: export
+  name: exporters
+  description: "Exporter Integrations"
+  most_popular: true
+  priority: 4
+  children:
+- id: notify
```
Comment (andrewm4894) [inline integrations/categories.yaml:410]: @sashwathn does this need to be indented?

**Comment by andrewm4894** [comment]
not sure the lists in here look correct

**Comment by andrewm4894** [comment]
eg i think this error is real: https://github.com/netdata/netdata/commit/156710085761cf4a0266b432b9fc929961f0445b#annotation_12921129506

**Review by shyamvalsan — APPROVED** [review]

**Comment by andrewm4894** [comment]
@shyamvalsan do you not see the yamlint error? i think its maybe a real one.

**Comment by andrewm4894** [comment]
checking to see if yamlint still complains

**Review by MrZammler — APPROVED** [review]

**Review by andrewm4894 — APPROVED** [review]

**Comment by andrewm4894** [comment]
hmm i merged since this yamlint passed - not sure whats going on i think its a missing ` []`

File: integrations/categories.yaml (hunk 2 above)
Comment (sashwathn) [reply integrations/categories.yaml:410]: I copied from the sections above but let me check.

**Pull request merged**